mugoh/qstioner
# #163017671 Display Questions to existing Meetups

#### What does this PR do?
Fetch and display Meetup Questions

#### Description of Task to be completed?
- A newly created question is visible on pages that show present questions.
- Show comments to meetup questions
- Update the page for displaying meetup questions to show more meetup elements.

#### How should this be manually tested?
Ensure the server is running. Follow the steps given [here](https://github.com/hogum/qstioner-v2) to do so.
1. Clone the repo in your machine
``` shell
$ git clone https://github.com/hogum/qstioner
```
2. Navigate to the directory `qstioner` where you have the repo cloned
``` shell
$ cd qstioner
```
3. Checkout this Feature branch
``` shell
$ git checkout ft-show-questions-163017671
```
4. Open the `index.html` file
``` shell
$ browser-name index.html
```
or
Simply  load up on your preferred browser
5. Navigate to the sign in page and sign in with account credentials.
- email: `admincow@mammals.milk` password: `pa555word`

6. Create a meetup on the admin dashboard.

7. Open  this meetup by clicking on it's title. A page opens where questions can be added to the meetup.
 Created meetups should be visible on the meetups display area.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#163017671](https://www.pivotaltracker.com/story/show/163017671)
